networking/cidr-range-definitions.adoc

```diff
@@ -35,12 +43,31 @@ endif::[]
 
 [id="service-cidr-description"]
 == Service CIDR
-In the Service CIDR field, you must specify the IP address range for services. It is recommended, but not required, that the address block is the same between clusters. This will not create IP address conflicts. The range must be large enough to accommodate your workload. The address block must not overlap with any external service accessed from within the cluster. The default is `172.30.0.0/16`.
+In the Service CIDR field, you must specify the IP address range for services. 
+ifdef::openshift-rosa,openshift-dedicated[]
+It is recommended, but not required, that the address block is the same between clusters. This will not create IP address conflicts. 
+endif::openshift-rosa,openshift-dedicated[]
+The range must be large enough to accommodate your workload. The address block must not overlap with any external service accessed from within the cluster. The default is `172.30.0.0/16`.
 
 [id="pod-cidr-description"]
 == Pod CIDR
-In the pod CIDR field, you must specify the IP address range for pods. It is recommended, but not required, that the address block is the same between clusters. This will not create IP address conflicts. The range must be large enough to accommodate your workload. The address block must not overlap with any external service accessed from within the cluster. The default is `10.128.0.0/14`.
+In the pod CIDR field, you must specify the IP address range for pods. 
+
+ifdef::openshift-enterprise[]
+The pod CIDR is the same as the `clusterNetwork` CIDR and the cluster CIDR. For more information about configuring CIDR for these, see xref:../networking/cluster-network-operator.adoc#nw-operator-cr-cno-object_cluster-network-operator[Cluster Network Operator Configuration] and the xref:../networking/configuring-cluster-network-range.adoc#configuring-cluster-network-range[Configuring the cluster network range]. 
```

Comment:
@abhat I added more info here to relate `clusterNetwork` CIDR and cluster CIDR to the pod CIDR definition.